anyway/parsers/registered.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+import csv
+import glob
+import io
+import logging
+import os
+import re
+import sys
+from datetime import datetime
+
+import six
+from flask.ext.sqlalchemy import SQLAlchemy
+
+from ..models import RegisteredVehicle, City
+from ..utilities import init_flask, time_delta
+
+reload(sys)
+sys.setdefaultencoding('utf8')
+
+# Headless servers cannot use GUI file dialog and require raw user input
+fileDialog = True
+try:
+    import tkFileDialog
+except (ValueError, ImportError):
+    fileDialog = False
+
+app = init_flask()
+db = SQLAlchemy(app)
+
+_manual_assing_on_city_name = {
+    'הרצלייה': 'הרצליה',
+    'תל אביב-יפו': 'תל אביב -יפו',
+    'אפרתה': 'אפרת',
+    "באקה-ג'ת": 'באקה אל-גרביה',
+    'בוקעאתה': 'בוקעאתא',
+    "ג'ש (גוש חלב)": "ג'ש )גוש חלב(",
+    'דבורייה': 'דבוריה',
+    'טובא-זנגרייה': 'טובא-זנגריה',
+    'יהוד': 'יהוד-מונוסון',
+    'יהוד-נווה אפרים': 'יהוד-מונוסון',
+    "כאוכב אבו אל- היג'א": "כאוכב אבו אל-היג'א",
+    "כעביה-טבאש- חג'אג'רה": "כעביה-טבאש-חג'אג'רה",
+    'מודיעין-מכבים- רעות' : 'מודיעין-מכבים-רעות',
+    'נהרייה' : 'נהריה',
+    "פקיעין (בוקייעה)" : "פקיעין )בוקייעה(",
+    'פרדסייה' : 'פרדסיה',
+    'קציר-חריש' : 'קציר',
+    'שבלי-אום אל-גנם' : 'שבלי - אום אל-גנם',
+}
+
+
+class DatastoreImporter(object):
```

Comment:
I think that this whole thing should be a part of `cbs.py`

Reply:
cbs load files that are related and should load as one, this is not the case in this import - each file is separated (I agree that the directory structure should be the same as in year/files but this is something that i will create and not something LMS bring as part of there drop - i will modify the structure)

*for my opinion the hole processing procedure should be OOP but I am new in the project :) - may be i am wrong because of it, or do not see something.

Reply:
I agree that if the scripts are independent then they don't have to be in the same file, but as much as possible should be shared as long as it's still clear.

Reply:
the scripts are independent, cbs work on csv with title on first line, this is not the case in here.